less/index.less

```diff
@@ -4,56 +4,401 @@
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html,
+body {
+  height: 100%;
+  font-size: 62.5%;
+  font-family: 'Lato', sans-serif;
 }
 
+// Variables
+@boxgray: rgb(156, 154, 154);
+@gray: rgb(150, 134, 134);
+@rem18: 1.8rem;
+@rem16: 1.6rem;
+
+//Mixins
+.center {
+  max-width: 980px;
+  margin: 0 auto;
+}
+.margin3tl {
+  margin-top: 3%;
+  margin-left: 3%;
+}
+.flex-class {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.nav-prop-mixin {
+  color: black;
+  font-size: @rem16;
+  text-decoration: none;
+}
+// Device widths
+// @phone: ~'(max-width:500px)';
+@phone: 500px;
+
+// END
+
+body {
+  .wrapper {
+    max-width: 980px;
+    .center();
+    line-height: 1.2;
+    background: #fff;
+    @media (max-width: 500px) {
+      text-align: center;
+      margin-right: 3%;
+    }
+  }
+  header {
+    margin-top: 2%;
+    margin-bottom: 4%;
+
+    nav {
+      display: flex;
+      justify-content: flex-end;
+      width: 100%;
+      @media (max-width: @phone) {
+        justify-content: center;
+      }
+      ul {
+        display: flex;
+        justify-content: space-between;
+        width: 40%;
+        flex-wrap: wrap;
+
+        @media (max-width: @phone) {
+          width: 100%;
+          flex-flow: column;
+          text-align: center;
+        }
+
+        li a {
+          text-decoration: none;
+          font-size: @rem16;
+          font-weight: bold;
+          color: black;
+        }
+      }
+    }
+  }
+  .main-container {
+    justify-content: space-between;
+    margin-right: 3%;
+    background: @boxgray;
+    width: 100%;
+    display: flex;
+    flex-flow: column;
+    padding-bottom: 7%;
+    @media (max-width: @phone) {
+      .flex-class();
+      margin-left: 1%;
+    }
+
+    .main-photo {
+      margin-top: 7%;
+      margin-left: 3%;
+      margin-right: 3%;
+      @media (max-width: @phone) {
+        img {
+          clip-path: polygon(0% 0%, 100% 3%, 98% 99%, 3% 93%);
+        }
+      }
+    }
+    .h1-container {
+      max-width: auto;
+      margin-left: 3%;
+      @media (max-width: @phone) {
+        margin-right: 3%;
+      }
+      h1 {
+        margin-top: 1%;
+        font-size: 5.8rem;
+      }
+      h2 {
+        font-size: 2.3rem;
+        line-height: 2.5;
+        color: rgb(28, 0, 131);
+      }
+      button {
+        font-size: 1.3rem;
+        color: white;
+        background: #808080;
+        border-radius: 5px;
+        padding: 1%;
+        margin-top: 2%;
+        &:hover {
+          background: lighten(#808080, 30%);
+          color: darken(white, 90%);
+        }
+      }
+    }
```

Comment:
I really like what you did with this button. The use of lighten and darken is excellent.